package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "zencoder",
-  "version": "1.1.0",
+  "version": "2.0.1",
```

Comment:
For what it's worth, this feels like it _could technically_ be a minor bump if you wanted to. As far as I can tell no current functionality is broken (👍 ), so it shouldn't break anything for folks that are pinned to `1.x`.

Reply:
I went with major since it will no longer work with node 0.1.0, which was the original version requirement. Wanted to use native promises which require 6.4 or above.

I'm open to doing a minor if you'd prefer it though.

Reply:
That reasoning makes total sense to me 👍